schemas/iso19139/src/main/plugin/iso19139/present/metadata-utils.xsl

```diff
@@ -87,6 +87,46 @@
     </xsl:call-template>
   </xsl:template>
 
+  <!-- iso3code from the supplied gmdlanguage
+       It will prefer LanguageCode if it exists over CharacterString -->
+  <xsl:template name="getLangIdFromGmdlanguage">
```

Comment:
Can't be changed to use `langId_from_gmdlanguage19139`?

Reply:
I did noticed the duplication and thought of making it just call one however I was not sure of the dependencies - and I did not want to cause any conflict issues.  I believe we should be able to remove these templates and just use the function version as long as it does not cause any conflicts.  I can try it to see if it causes any noticeable issues.

Reply:
I have updated the code to remove duplication